server/lyft/temporal/activities/terraform.go

```diff
@@ -0,0 +1,97 @@
+package activities
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+type InitRequest struct {
+	RootDir string
+}
+
+type InitResponse struct {
+	Output string
+}
+
+type PlanRequest struct {
+	RootDir string
+}
+
+type PlanResponse struct {
+	Output   string
+	Planfile string
+}
+
+type ApplyRequest struct {
+	RootDir  string
+	Planfile string
+}
+
+type ApplyResponse struct {
+	Output string
+}
+
+func Init(ctx context.Context, request InitRequest) (InitResponse, error) {
+	output, err := terraform(request.RootDir, "init", "-input=false")
+
+	// output using fmt instead of logger to get pretty formatting.
+	// this is probably susceptible to duplication though in the event of replays
+	// this is only for prototype purposes anyways.
+	fmt.Println(string(output))
```

Comment:
You could use `if !workflow.IsReplaying(ctx)` to avoid the duplicate log during replay. 

This one of the reasonable use-cases they call out directly:
https://github.com/temporalio/sdk-go/blob/2c5ed0f2b1424c44852e144b7137ba424b33095f/workflow/workflow.go#L398